docs/en_us/internal/testing.rst

```diff
@@ -94,6 +94,22 @@ Test Locations
    -  Bok Choy Accessibility Tests: located under ``common/test/acceptance/accessibility``
    -  Bok Choy PageObjects: located under ``common/test/acceptance/pages``
 
+-  Content type examples:
+
+   The Accessibility Mega Course
+   `https://github.com/cptvitamin/demo-test-course
+   <https://github.com/cptvitamin/demo-test-course>`_ provides
```

Comment:
@pwilkins can you update this to the new permanent home for this course: <https://github.com/edx/demo-test-course>